includes/Story_Renderer.php

```diff
@@ -133,6 +133,106 @@ protected function add_poster_images( $content ) {
 		return $content;
 	}
 
+	/**
+	 * Print amp-analytics script.
+	 *
+	 * @return void
+	 */
+	public function print_analytics_script() {
+		?>
+		<script async="async" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js" custom-element="amp-analytics"></script>
+		<?php
+	}
+
+	/**
+	 * Prints AMP Analytics based on Site Kit configuration.
```

Comment:
Can you add a link here to where you copied this from? Makes it easier to update in the future.

Reply:
Just updated it based on the configuration needed for Stories based on [this doc](https://docs.google.com/document/d/1q8rkYYMKZh3R2eqkpwgGkx6QX0oSXoLYtIO6ml1BFD0/edit#).

Reply:
(Meaning that now the configuration is not copied from Site Kit anymore.)